Scripts/fastlane/Fastfile

```diff
@@ -1,11 +1,18 @@
 default_platform(:ios)
 fastlane_require 'dotenv'
 
+before_all do
+  # Check that the env files exist
+  UI.user_error!("env/user.env not found: Please copy env/user.env-example to env/user.env and fill in the values") unless File.file?('env/user.env')
+  UI.user_error!("env/project.env not found: Make sure your configuration is up to date with `bundle exec fastlane run configure_apply`") unless File.file?('env/project.env')
+end
+
 platform :ios do
 ########################################################################
 # Environment
 ########################################################################
-Dotenv.load('~/.wpios-env.default')
+Dotenv.load('env/user.env')
```

Comment:
Note that the user env file now lives in the repo and not at `~/.wpios-env.default`. If anyone has a preference, I am open to keeping it at the old location. Having it in the repo is more consistent with everything else we do.

Reply:
To be honest, I don't like this solution a lot. I mean, it's not specifically related to _this_ file.
I see we have added other secrets in the repo and I agree that we have to be consistent with the layout, so I wonder if there's a technical reason to handle all these files in the repo or if they could be places outside without adding technical efforts. 
With the files in the repo, there's always the risk that they get committed in an older branch that doesn't .gitignore them.

Reply:
Thanks for your comment @loremattei! Those are all fair points. Lets chat about this later 👍

Reply:
We discussed this offline and agreed to move all sensitive files to one directory to mitigate the issue. I have opened a PR for that in https://github.com/wordpress-mobile/WordPress-iOS/pull/12668. Once that is merged, I will update this PR to do the same.